spec/classes/foreman_proxy_content_spec.rb

```diff
@@ -71,6 +71,39 @@ class { 'foreman_proxy::plugin::pulp':
         end
       end
 
+      context 'pulp-2to3-migration' do
+        let(:params) do
+          {
+            qpid_router: false
+          }
+        end
+
+        let(:pre_condition) do
+          <<-PUPPET
+          include foreman_proxy
+          class { 'foreman_proxy::plugin::pulp':
+            enabled          => true,
+            pulpnode_enabled => false,
+            pulpcore_enabled => true,
+            pulpcore_mirror  => false,
+          }
+          PUPPET
+        end
+
+        it { is_expected.to compile.with_all_deps }
+        it { is_expected.to contain_class('pulp').with(manage_squid: true) }
+        it { is_expected.to contain_class('pulpcore').with(manage_apache: false) }
```

Comment:
I'm not clear how this test and the test above it are related to the migration code.

Reply:
@ehelms they aren't. My thinking is that we might as well test that pulp and pulpcore continue to work in the case where both are present and we are turning the migration plugin on. In fact since puppet-katello handles pulp setup for the main katello server, I should definitely remove the pulp test here. I can remove the pulpcore test as well if you feel it is redundant.